operator-metadata/automation/tests/test_bundle_automation.py

```diff
@@ -82,11 +83,11 @@ def test_update_new_csv_format(self):
             csv_file_path=self.NEW_CSV_INTERNAL_PULL_SPECS_FILE_PATH,
             bundle_versions=["2.4.0-0", "2.5.0-0"],
             replacement_map={
-                "strimzi-rhel9-operator:2.5.0-1": self.OPERATOR_PULL_SPEC_REPLACEMENT,
-                "kafka-34-rhel9:2.5.0-1": self.KAFKA_PREVIOUS_PULL_SPEC_REPLACEMENT,
-                "kafka-35-rhel9:2.5.0-1": self.KAFKA_CURRENT_PULL_SPEC_REPLACEMENT,
-                "bridge-rhel9:2.5.0-1": self.BRIDGE_PULL_SPEC_REPLACEMENT,
-                "maven-builder-rhel9:2.5.0-1": self.MAVEN_PULL_SPEC_REPLACEMENT
+                "strimzi-rhel8-operator:2.5.0-1": self.OPERATOR_PULL_SPEC_REPLACEMENT,
```

Comment:
Re. ` test_update_new_csv_format`  is maintaining the `old` way really required?

Reply:
> Less code = less to effort to maintain.

Agreed

But I think we should keep/maintain the old way at least until we confirm the Freshmaker automation behaves as expected. If we need to revert from the new way to the old way it would good to have some unit tests in place

Reply:
> But I think we should keep/maintain the old way at least until we confirm the Freshmaker automation behaves as expected. If we need to revert from the new way to the old way it would good to have some unit tests in place

That's SCM's job, isn't it?    If we still had a way to pull a switch (without changing code) and get back the code behaviour, then I might agree with you.  But as things are we'd need to be make a code change to go back.  So we may as will kill the old code now.  Less code = less mental load for any developer who maintains the it.

Reply:
> That's SCM's job, isn't it? If we still had a way to pull a switch (without changing code) and get back the code behaviour, then I might agree with you. But as things are we'd need to be make a code change to go back. So we may as will kill the old code now. Less code = less mental load for any developer who maintains the it.

That's a fair point. Could we cover this in a separate PR?